modules/client-common/src/constants.ts

```diff
@@ -0,0 +1,124 @@
+import { activeContractsList } from "@aragon/osx-ethers";
```

Comment:
Are we removing the original files? I almost didn't see any removal and this would bring lots of code duplication